tests/qa_automation/qa_run.pm

```diff
@@ -87,7 +89,8 @@ sub prepare_repos {
     }
     # sometimes updates.suse.com is busy, so we need to wait for possiblye retries
     zypper_call("--gpg-auto-import-keys ref");
-    zypper_call("in qa_testset_automation qa_tools");
+    add_suseconnect_product('sle-module-python2') if is_sle('>15');
+    zypper_call("in qa_testset_automation qa_tools python-base python-xml");
```

Comment:
I think python-xml is not needed on this line, it will be pulled as dependency(as recommends) by qa_testset_automation.

Reply:
No, that did not work: https://openqa.suse.de/tests/2492872/file/serial_terminal.txt

Reply:
Ok. so I'm fine with original proposal,